apps/central-scan/frontend/src/app_root.tsx

```diff
@@ -472,11 +465,7 @@ export function AppRoot({
   }
 
   if (electionDefinition) {
-    if (
-      wasJustConfigured &&
-      configureMutation.data.isOk() &&
-      !configureSuccessScreenClosed
-    ) {
+    if (configureMutation.isSuccess && configureMutation.data.isOk()) {
```

Comment:
Is this check redundant? Should it be impossible to have an `electionDefinition` without a successful configure?

Reply:
Not redundant, because you could have configured the machine last time you powered on or before we reset the mutation. Will be reworking this to be more similar to VxScan anyway, though, so this logic will likely be removed.